src/webgpu/shader/execution/expression/call/builtin/dot.spec.ts

```diff
@@ -118,12 +118,6 @@ g.test('u32_vec4')
     await run(t, builtin('dot'), [Type.vec4u, Type.vec4u], Type.u32, t.params, cases);
   });
 
-g.test('abstract_float')
```

Comment:
Which test covers abstract_float? Can we update the relevant test name to make it clearer?

Reply:
Apparently I am really good at not including new test boilerplate in my PRs... I have added the missing tests